python-backend/app/api/permits/permit_amendment/models/permit_amendment.py

```diff
@@ -72,29 +102,26 @@ def validate_permit_id(self, key, permit_id):
 
     @validates('received_date')
     def validate_received_date(self, key, received_date):
-        if not received_date:
-            return received_date
-        if received_date.isoformat() == '9999-12-31':
-            raise AssertionError(
-                'Permit amendment received date should be set to null if not known.')
-        if received_date > datetime.today():
-            raise AssertionError('Permit amendment received date cannot be set to the future.')
+        if received_date:
+            if received_date.isoformat() == '9999-12-31':
+                raise AssertionError(
+                    'Permit amendment received date should be set to null if not known.')
+            if received_date > datetime.today():
+                raise AssertionError('Permit amendment received date cannot be set to the future.')
         return received_date
 
     @validates('issue_date')
     def validate_issue_date(self, key, issue_date):
-        if not issue_date:
-            return issue_date
-        if issue_date.isoformat() == '9999-12-31':
-            raise AssertionError('Permit amendment issue date should be set to null if not known.')
-        if issue_date > datetime.today():
-            raise AssertionError('Permit amendment issue date cannot be set to the future.')
+        if issue_date:
+            if issue_date.isoformat() == '9999-12-31':
```

Comment:
Also here. Why refactor away the early return? This was creates deeper conditional nesting while appearing to have the same behaviour